public/app/features/query-library/api/endpoints.gen.ts

```diff
@@ -0,0 +1,486 @@
+import { queryLibraryApi as api } from './factory';
+export const addTagTypes = ['QueryTemplate'] as const;
+const injectedRtkApi = api
+  .enhanceEndpoints({
+    addTagTypes,
+  })
+  .injectEndpoints({
+    endpoints: (build) => ({
+      listQueryTemplate: build.query<ListQueryTemplateApiResponse, ListQueryTemplateApiArg>({
```

Comment:
Btw, you can make these type names a bit shorter, by [configuring](https://github.com/grafana/grafana/pull/98891/files#diff-74cb865c9ea5f7944c361ccfa939ecc82519a3776eef960b412e5551ef5388deR44-R45) their suffixes.